ui-kit/src/main/kotlin/org/p2p/uikit/UiKitEditTextStyleAttrsApplier.kt

```diff
@@ -0,0 +1,59 @@
+package org.p2p.uikit
+
+import androidx.core.content.res.getDrawableOrThrow
+import androidx.core.content.res.getIntOrThrow
+import androidx.core.content.res.getResourceIdOrThrow
+import androidx.core.view.isVisible
+import android.content.res.TypedArray
+import android.text.method.DigitsKeyListener
+import org.p2p.uikit.databinding.WidgetUiKitEdittextBinding
+
+internal class UiKitEditTextStyleAttrsApplier {
```

Comment:
it's a pitty that [airbnb/paris](https://github.com/airbnb/paris) is not usable anymore with the new gradle and android policy